cloudmarker/clouds/gcpcloud.py

```diff
@@ -37,29 +39,39 @@ def __init__(self, key_file_path, _max_projects=0):
         Arguments:
             key_file_path (str): Path of the service account
                 key file for a project.
+            processes (int): Number of processes to launch.
+            threads (int): Number of threads to launch in each process.
             _max_projects (int): Maximum number of projects to fetch
                 data for if the value is greater than 0.
         """
         self._key_file_path = key_file_path
+        self._processes = processes
+        self._threads = threads
         self._max_projects = _max_projects
-        # Generating scoped credentials which will be required by the
-        # discovery.build to create a resource object which we will use to
-        # communicate with the API.
-        credentials = service_account.Credentials
-        self._credentials = credentials.from_service_account_file(
-            self._key_file_path,
-            scopes=_GCP_SCOPES)
 
-        # Currently generating scoped resources with googleapiclient, requires
-        # service_name and version information, the more complete information
-        # about available_service name and their corresponding version is
-        # located at
-        # https://developers.google.com/api-client-library/python/apis/
-        self._compute_resource = self._get_resource('compute', 'v1')
+        # Service account key file also has the client email under the key
+        # client_email. We will use this key file to get the client email for
+        # this request.
+        with open(self._key_file_path) as f:
+            self._client_email = json.loads(f.read())['client_email']
```

Comment:
Can you change this to `json.loads(f.read()).get('client_email')`, so that even if `'client_email'` key is missing, this line of code would not crash the plugin?